import-export-cli/docs/apictl_import_policy.md

```diff
@@ -13,8 +13,8 @@ apictl import policy [flags]
 ### Examples
 
 ```
-apictl import policy rate-limiting -f ~/CustomPolicy -e production --u
-apictl import policy api -f ~/AddHeader -e production
+apictl import policy rate-limiting -f ~/CustomPolicy -e production -u
+apictl import policy api  -f ~/AddHeader -e production
```

Comment:
can we get rid of this extra white space before `-f`?

Reply:
fixed via https://github.com/wso2/product-apim-tooling/pull/961/commits/a00e6e265ef72b3e7a9be54d43b92d9d5c5951cb